benchmarks/benchmarks.py

```diff
@@ -47,6 +47,9 @@ class LargeSimulationBenchmark:
     rounds = 1
     min_run_count = 1
     timeout = 120
+    # As we may wish to change code but not have ASV reset benchmarks we set a
+    # fixed version.
+    version = "1"
```

Comment:
I wasn't sure about this - does this mean we have a single version for all the benchmarks in this class? So, if we change one do we then invalidate all the benchmarks? If we do tweak the code, isn't it most likely to be to change parameters? I think we'd be better off using ASV's approach until we're sure that it doesn't work and resort to manual versioning of each individual benchmark as a last resort.

Reply:
I added this in before I realised it was no bother to regenerate past benchmarks, I've removed it now.